pyrdp/core/settings.py

```diff
@@ -0,0 +1,32 @@
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2020 GoSecure Inc.
+# Licensed under the GPLv3 or later.
+#
+from configparser import ConfigParser, ExtendedInterpolation
+
+import appdirs
+CONFIG_DIR = appdirs.user_config_dir("pyrdp", "pyrdp")
+
+
+def load(path: str, fallback: str) -> ConfigParser:
+    """
+    Retrieve the PyRDP settings from a file
+
+    :param path: The path of the file to load.
+    :param fallback: The fallback configuration.
+
+    :returns: A dict of the settings or None if the file was not present.
```

Comment:
I don't see how this function can ever return None, also type hints state that it returns a `ConfigParser`. Maybe this docstring is outdated with current implementation?

Reply:
docstring is outdated from when I was iterating, nice catch.